app/src/renderer/node.js

```diff
@@ -61,6 +62,28 @@ module.exports = function (nodeIP) {
 
       node.rpcConnecting = false
       return nodeIP
+    },
+    async sendTx (args, account, password) {
+      let tx = await (async function () {
+        switch (args.type) {
+          case 'buildDelegate': return axios.post('http://localhost:8998/build/stake/delegate', args)
+            .then(res => res.data)
+          case 'buildUnbond': return axios.post('http://localhost:8998/build/stake/unbond', args)
+            .then(res => res.data)
+          default: return node[args.type](args)
+        }
+      })()
+      let signedTx = await node.sign({
+        name: account,
+        password: password,
+        tx
+      })
+      let res = await node.postTx(signedTx)
```

Comment:
It's important that this goes through the same wallet sending code that it was separated from in this PR, to properly process multiple transactions serially and set the correct account sequence number on each transaction. Otherwise, transactions created here will only suceed if it is an account's first transaction.